Dockerfile

```
@@ -1,5 +1,5 @@
 # Dockerfile to create a go-getter container with smbclient dependency that is used by the get_smb.go tests
-FROM golang:1.15
+FROM golang:1.19.13
```

Would using a newer version of go based of an alpine one (eg: `1.22.4-alpine3.20`) work here?

```suggestion
FROM golang:1.22.4-alpine3.20
```

---

Not sure we can try. I opted for 1.19 because our tests run with 1.19 and 1.20. If it is not relevant to the fix. I would rather open a separate PR updating versions of Go used on CI.

---

That sounds good 👍  we can tackle it in a separate PR